index.html

```diff
@@ -13,13 +13,2689 @@
       href="https://fonts.googleapis.com/css2?family=Roboto+Condensed:wght@300;400;700&display=swap"
       rel="stylesheet"
     />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
+    />
     <link rel="stylesheet" href="assets/css/custom.css" />
+
     <title>Newsweek</title>
   </head>
 
-  <body>
-    <header></header>
-    <main></main>
-    <footer></footer>
+  <body class="container-fluid px-0">
+    <header class="container-fluid max-auto mx-0 px-0">
```

Comment:
Please make the `header` fixed in tablet and mobile screens.

| Source Page	| Clone Page |
|:---:|:---:|
|![Annotation 2020-07-11 160158](https://user-images.githubusercontent.com/57726348/87222286-329bce00-c390-11ea-9892-69f3a34ddf4d.jpg)|![Annotation 2020-07-11 160222](https://user-images.githubusercontent.com/57726348/87222294-3b8c9f80-c390-11ea-8ea5-4daf0e82afad.jpg)|